example_project/example_project/settings.py

```diff
@@ -124,3 +124,5 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+# WARNING: Do not set AETOS_* settings, otherwise tests fail.
```

Comment:
Line endings are inconsistent in this PR, did you activate the pre-commit hooks?

Reply:
Activated pre-commit hooks now 1ca452cc1ac1a01b440639f76510e6c3e4813c12